CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+v1.7.0
+------
+* **New features:**
+    * Add a new command that 'prunes' the tiles of interest set based on various rules. See [#176](https://github.com/tilezen/tilequeue/pull/176), [#178](https://github.com/tilezen/tilequeue/pull/178), [#179](https://github.com/tilezen/tilequeue/pull/179), [#180](https://github.com/tilezen/tilequeue/pull/180), [#182](https://github.com/tilezen/tilequeue/pull/182), [#183](https://github.com/tilezen/tilequeue/pull/183), [#184](https://github.com/tilezen/tilequeue/pull/184), and [#189](https://github.com/tilezen/tilequeue/pull/189).
```

Comment:
instead of 'prune' let's say 'garden', and since that's colloquial let's clarify with:

`Add new command that 'gardens' the tiles of interest set to add and remove tiles based on various rules.`